### ISSUE TITLE
Keyboard shortcuts collide with writing polish letters (Alt + letter)

### ISSUE THREAD

**Issue by pl86** [zadam/trilium#426]
App version: | 0.30.1-beta

Some shortcuts collide with the way polish letters (ą,ć,ę,ł,ń,ó,ś,ź,ż) are entered via right **Alt + normal letter** or capital letters via **Alt + Shift + normal letter**.

List of all keyboard shortcuts for polish letters:
```
Alt + a, Alt + Shift + a
Alt + c, Alt + Shift + c
Alt + e, Alt + Shift + e
Alt + l, Alt + Shift + l
Alt + n, Alt + Shift + n
Alt + o, Alt + Shift + o
Alt + s, Alt + Shift + s
Alt + x, Alt + Shift + x
Alt + z, Alt + Shift + z
```

I found this in file: `src/public/javascripts/services/entrypoints.js`:

```JavaScript
    utils.bindShortcut('alt+a', attributesDialog.showDialog);
    utils.bindShortcut('alt+o', sqlConsoleDialog.showDialog);
```

***Alt + c*** is defined as a shortcut too and collapse tree view

***Alt + n*** write "ń" letter in note and add new Task note in the same time

**Comment by zadam** [comment]
That's a pickle. I guess the only real solution is to let the user define their own keyboard shortcuts in such cases ...

**Comment by pl86** [comment]
Configuration file with shortcuts like `keybindings.json` in Visual Studio Code would be great.

I found other common shortcut i use a lot ***Cmd + left/right*** on Mac to move between beginning and end of current line. In trillium it moves to previous/next note.

Only those shortcuts stops me from moving all my notes to trillium.

**Comment by expelledboy** [comment]
@pl86 I think so too, heres a [first draft](https://github.com/zadam/trilium/pull/675).

**Comment by zadam** [comment]
customizable keyboard shortcuts have been implemented in 0.38
